activemodel/lib/active_model/validations/acceptance.rb

```diff
@@ -3,12 +3,12 @@ module ActiveModel
   module Validations
     class AcceptanceValidator < EachValidator # :nodoc:
       def initialize(options)
-        super({ allow_nil: true, accept: "1" }.merge!(options))
+        super({ allow_nil: true, accept: ["1", true] }.merge!(options))
```

Comment:
I wonder if that would lead to a possible explosion of options. i.e. "True", "TRUE"

Currently, if you need to add additional accept options you can do so.

```ruby
class User < ActiveRecord::Base
  validates_acceptance_of :terms_and_conditions, accept: ["True", "TRUE", "true"]
end
```

I would like to keep this change minimal, as I believe TrueClass should be one of default accept options.